pkg/shell/topnav.tsx

```diff
@@ -77,12 +77,11 @@ export class TopNav extends React.Component {
             theme: localStorage.getItem('shell:style') || 'auto',
         };
 
-        read_os_release().then(os => this.setState({ osRelease: os || {} }));
-
         this.handleClickOutside = () => this.setState({ menuOpened: false, docsOpened: false });
     }
 
     componentDidMount() {
+        read_os_release().then(os => this.setState({ osRelease: os || {} }));
```

Comment:
We don't have to do this at all any more since last week. But let's fix this up wholesale.

Reply:
We still have to wait on `cockpit.info()` somehow

Reply:
Yes, but that should happen at the very top-level entry point. Like we do in files now. https://github.com/cockpit-project/cockpit-files/blob/main/src/index.tsx#L40